apisix/cli/ops.lua

```diff
@@ -390,6 +392,21 @@ local function start(env, ...)
         end
     end
 
+    local parser = argparse()
+    parser:argument("_", "Placeholder")
+    parser:option("-c --config", "location of customized config.yaml")
+    local args = parser:parse()
+    local customized_yaml = args["config"]
+
+    profile.apisix_home = env.apisix_home .. "/"
+    local local_conf_path = profile:yaml_path("config")
+
+    if customized_yaml then
+        execute("mv " .. local_conf_path .. " " .. local_conf_path .. ".bak")
```

Comment:
we need to revert the file `con/config.yaml.bak` when call `apisix stop`

Reply:
and we need the test cases about this step

Reply:
Sure, working on it

Reply:
PTAL